### PR TITLE
Initial conversion to Fedora Bootstrap

### PR DESCRIPTION
**Pull request by ryanlerch**
This changes the master template to use the Fedora Bootstrap
styles. Other UX and pages are unchanged, mainly just a chrome
change to make it similar to all the other apps

### CODE REVIEW

File: mirrormanager2/templates/fedora/master_noauth.html [hunk 1]
```diff
@@ -5,47 +5,52 @@
...
         href="{{ url_for('static', filename='favicon.ico')}}"/>
-    <link rel="stylesheet" type="text/css" media="screen"
-        href="{{ url_for('static', filename='koji.css') }}"/>
+        <link href="https://apps.fedoraproject.org/global/fedora-bootstrap-1.0.2/fedora-bootstrap.css"
+              type="text/css" rel="stylesheet" />
     <link rel="stylesheet" type="text/css" media="screen"
         href="{{ url_for('static', filename='mirrormanager2.css') }}"/>
+    <link href="https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/hack.css" type="text/css" rel="stylesheet" />
+    <link href="https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/open-sans.css" type="text/css" rel="stylesheet" />
+    <link href="https://ryanlerch.fedorapeople.org/font-awesome.css" type="text/css" rel="stylesheet" />
```
Comment (bowlofeggs) [inline mirrormanager2/templates/fedora/master_noauth.html]: Should this be served from apps.fpo as well?

Comment (ryanlerch) [inline mirrormanager2/templates/fedora/master_noauth.html]: yeah, i'll change it back, sorry!

Just waiting on a patch that has been sitting in the infra queue for a while that makes this font work for local testing too.

**Comment by adrianreber** [comment]
@ryanlerch if you rebase and fix the link I can merge this PR.

**Comment by ryanlerch** [comment]
@adrianreber done, rebased and changed the link to the fontawesome CSS hosted on apps.fp.org/global

**Comment by adrianreber** [comment]
@ryanlerch Thanks for the rebase. I will merge it thanks to @bowlofeggs review and include it in the next release.

**Pull request merged**